openzwave-sys/src/notification.rs

```diff
@@ -56,6 +56,52 @@ impl fmt::Display for NotificationCode {
     }
 }
 
+c_like_enum! {
+    ControllerState {
+        ControllerStateNormal = 0,     // No command in progress.
+        ControllerStateStarting = 1,   // The command is starting.
+        ControllerStateCancel = 2,     // The command was cancelled.
+        ControllerStateError = 3,      // Command invocation had error(s) and was aborted.
+        ControllerStateWaiting = 4,    // Controller is waiting for a user action.
+        ControllerStateSleeping = 5,   // Controller command is on a sleep queue wait for device.
+        ControllerStateInProgress = 6, // The controller is communicating with the other device to carry out the command.
+        ControllerStateCompleted = 7,  // The command has completed successfully.
+        ControllerStateFailed = 8,     // The command has failed.
+        ControllerStateNodeOK = 9,     // Used only with ControllerCommand_HasNodeFailed to indicate that the controller thinks the node is OK.
+        ControllerStateNodeFailed = 10 // Used only with ControllerCommand_HasNodeFailed to indicate that the controller thinks the node has failed.
```

Comment:
I wonder if we need to keep the ControllerState prefix here; actually I think we shouldn't (and eventually remove the other prefixes we have too). It's not needed for C compatibility and it's makes things messy.

And it should be easy to `s/::ControllerState/::/g` in the code :)